src/components/file/file.tsx

```diff
@@ -235,8 +235,8 @@ export class File {
                 icon: getFileIcon(this.value),
                 iconFillColor: getFileColor(this.value),
                 iconBackgroundColor: getFileBackgroundColor(this.value),
-                href: this.value.href,
                 iconTitle: getFileExtensionTitle(this.value),
```

Comment:
I don't know how to make these work with the new Icon interface in a backwards compatible way. 

But also this one could be a breaking change if we want. The impact is super low. I don't think anyone have customized their icon's color in limel-file. They should be really "sick" if they have done so! 😅 And if they upgrade and get no colors, it won't be a big deal.

Reply:
@jgroth I'd also need help with this one, if you have time.

Reply:
Should it not just be something like this? :thinking: 
```
icon: {
    name: getFileIcon(this.value),
    title: getFileExtensionTitle(this.value),
    color: getFileColor(this.value),
    backgroundColor: getFileBackgroundColor(this.value),
},
```